cocos/tween/actions/action-interval.ts

```diff
@@ -338,49 +340,49 @@ export class Sequence extends ActionInterval {
      * @param {FiniteTimeAction} actionTwo
      * @return {Boolean}
      */
-    initWithTwoActions (actionOne: any, actionTwo: any): boolean {
+    initWithTwoActions (actionOne?: FiniteTimeAction, actionTwo?: FiniteTimeAction): boolean {
         if (!actionOne || !actionTwo) {
             errorID(1025);
             return false;
         }
 
         let durationOne = actionOne._duration; let durationTwo = actionTwo._duration;
-        durationOne *= actionOne._repeatMethod ? actionOne._timesForRepeat : 1;
-        durationTwo *= actionTwo._repeatMethod ? actionTwo._timesForRepeat : 1;
+        durationOne *= (actionOne as any)._repeatMethod ? actionOne._timesForRepeat : 1;
+        durationTwo *= (actionTwo as any)._repeatMethod ? actionTwo._timesForRepeat : 1;
```

Comment:
Why need to change to any here?

Reply:
Because `actionOne` is `FiniteTimeAction` type but _repeatMethod is a property inside `ActionInterval` class.

```ts
initWithTwoActions (actionOne?: FiniteTimeAction, actionTwo?: FiniteTimeAction)
```